Signum.Entities/FieldAttributes.cs

```diff
@@ -319,6 +319,7 @@ public sealed class PrimaryKeyAttribute : DbTypeAttribute
 
         public bool Identity { get; set; }
 
+        public bool SequentialGuid;

```

Comment:
Used somewhere? Looks like is controlled by DefaultSqlServer = SqlServer_NewSequentialId()

Reply:
Seems to be obsolete. I've removed that.